pkg/gateway/gateway.go

```diff
@@ -350,11 +352,30 @@ func (g *gatewayModule) isTraefikStarted(z *zinit.Client) (bool, error) {
 	return traefikStatus.State.Is(zinit.ServiceStateRunning), nil
 }
 
+func (g *gatewayModule) traefikBinary(ctx context.Context) (string, string, error) { // path, name, err
+	var exec struct{ Exec string }
+	f, err := os.Open("/etc/zinit/traefik.yaml")
+	if err != nil {
```

Comment:
you need to handle the IsNotExist error separately because this will get returned if the service is not configured. 

I also really think this has to be done through zinit as well. because the file can exist on the filesystem BUT zinit did never track it up (no zinit monitor was done)